USAGE.md

```diff
@@ -0,0 +1,48 @@
+# config-parse(1) -- screwdriver config parser
```

Comment:
Yeah, 1 refers to a user command: http://man7.org/linux/man-pages/man7/man-pages.7.html